### PR TITLE
Migrate Account Settings to PF4

### PR DESCRIPTION
**Pull request by rszwajko**
Depens on #1537

Includes:
1. new SelectBox
2. reactoring to link fieldId with label provided by FormGroup

### CODE REVIEW

**Comment by rszwajko** [comment]
### General settings
account_settings_general_pf4
### Refresh rate
account_settings_refresh_pf4
### Notifications
account_settings_notifications_pf4
### Console settings
account_settings_console_top_pf4
account_settings_console_bottom_pf4

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

File: src/components/SelectBox.js [hunk 1]
```diff
@@ -1,80 +1,64 @@
...
+      >
+        {options.map((option) => (
+          <SelectOption
+            key={option.id}
+            value={option}
+            description={option.isDefault ? msg.defaultOption() : undefined}
+          />
+        ))}
+      </Select>
+    </>
```
Comment (sjd78) [inline src/components/SelectBox.js:44]: nit-pick: I don't think the fragment is necessary

File: src/components/Settings/SettingsBase.js [hunk 2]
```diff
@@ -2,92 +2,92 @@ import React from 'react'
...
-        <LabelCol fieldPath={`${name}-${field.name}`} tooltip={field.tooltip} sm={3} className={style['field-label']}>
-          { field.title }
-        </LabelCol>
-        <Col sm={9}>
-          {field.body}
-        </Col>
-      </FormGroup>
+      <CardBody key={field.key}>
+        {
+        field.title && (
```
Comment (sjd78) [inline src/components/Settings/SettingsBase.js:32]: nit-pick: should be a single line

**Review by sjd78 — CHANGES_REQUESTED** [review]
The extra `propTypes` on `SettingsBase` look nice.
A handful of nit-picks, but looks real good.

**Comment by ovirt-infra** [comment]
All tests passed

**Review by sjd78 — APPROVED** [review]
LGTM

**Pull request merged**